src/Sylius/Bundle/CoreBundle/Doctrine/ORM/PaymentMethodRepository.php

```diff
@@ -24,7 +24,6 @@ public function createListQueryBuilder($locale)
     {
         return $this
             ->createQueryBuilder('o')
-            ->addSelect('translation')
```

Comment:
You're right! There are many methods without selects, we will have to add them...